classes/class-wc-connect-taxjar-integration.php

```diff
@@ -173,11 +173,20 @@ public function is_enabled() {
 	 * @return array
 	 */
 	public function add_tax_settings( $tax_settings ) {
-		$enabled = $this->is_enabled();
+		$enabled                = $this->is_enabled();
+		$backedup_tax_rates_url = admin_url( '/admin.php?page=wc-status&tab=connect#tax-rate-backups' );
+
+		$powered_by_wct_notice = '<p>' . __( 'Powered by WooCommerce Tax. If automated taxes are enabled, you\'ll need to enter prices exclusive of tax.', 'woocommerce-services' ) . '</p>';
+
+		if ( ! empty( WC_Connect_Functions::get_backed_up_tax_rate_files() ) ) {
+			$powered_by_wct_notice .= '<p>' . sprintf( __( 'Your previous tax rates were backed up and can be downloaded %1$shere%2$s.', 'woocommerce-services' ), '<a href="' . $backedup_tax_rates_url . '">', '</a>' ) . '</p>';
```

Comment:
```suggestion
			$powered_by_wct_notice .= '<p>' . sprintf( __( 'Your previous tax rates were backed up and can be downloaded %1$shere%2$s.', 'woocommerce-services' ), '<a href="' . esc_url( $backedup_tax_rates_url ) . '">', '</a>' ) . '</p>';
```
Adding an `esc_url()` for better URL sanitization.